projects/core/src/store-finder/facade/store-data.service.spec.ts

```diff
@@ -2,6 +2,13 @@ import { TestBed } from '@angular/core/testing';
 import { PointOfService } from '../../model/point-of-service.model';
 import { StoreDataService } from './store-data.service';
 
+const YEAR_2018 = 2018;
+const MONTH_AUGUST = 8;
+const DAY_SEVENTEEN = 17;
+const DAY_TWENTY_THREE = 23;
```

Comment:
you can move that to describe block and avoid of putting date in const name (and do it in that way https://github.com/SAP/spartacus/pull/8228/files#r458710140)

Reply:
Done